apps/yarn.lock

```diff
@@ -63,15 +63,9 @@
     firmata latest
     serialport "^4.0.0"
 
-"@code-dot-org/js-interpreter":
-  version "1.1.1"
-  resolved "https://registry.yarnpkg.com/@code-dot-org/js-interpreter/-/js-interpreter-1.1.1.tgz#2a7ccbb6b1f31353445f82f777573d1c38e11d37"
-  dependencies:
-    yargs "^7.0.1"
-
-"@code-dot-org/js-interpreter-tyrant":
-  version "0.1.1"
-  resolved "https://registry.yarnpkg.com/@code-dot-org/js-interpreter-tyrant/-/js-interpreter-tyrant-0.1.1.tgz#34153bc7620748f07c687849741bbd7ba3d420a7"
+"@code-dot-org/js-interpreter-tyrant@0.2.2":
+  version "0.2.1"
+  resolved "https://registry.yarnpkg.com/@code-dot-org/js-interpreter-tyrant/-/js-interpreter-tyrant-0.2.1.tgz#bf7f20cc8689a1059cef8bd5a6e25a024eec9c84"
```

Comment:
weird. You know what, let me double check this. It might be that I had used yarn link to point to a local checkout of js-interpreter-tyrant and that somehow screwed up the resolution.